src/AccessibilityInsights.SharedUx/Controls/EventRecordControl.xaml

```diff
@@ -117,26 +117,26 @@
                     <fabric:FabricIconControl GlyphName="Settings" GlyphSize="Custom" FontSize="12" HorizontalAlignment="Center" VerticalAlignment="Center" Foreground="{DynamicResource ResourceKey=IconBrush}"/>
                     <Button.ContextMenu>
                         <ContextMenu FlowDirection="LeftToRight" Loaded="ContextMenu_Loaded" Unloaded="ContextMenu_Unloaded">
-                            <MenuItem Header="Event recording scope" FontWeight="Bold" IsEnabled="False" IsCheckable="False"/>
-                            <MenuItem x:Name="mniRaw" IsCheckable="False" Click="mniRB_Click"
+                            <MenuItem Header="Event recording scope">
```

Comment:
This should probably be in resources

Reply:
I realize this was already not in resources, so if you want I can let it through